src/main/groovy/com/auth0/gradle/oss/AndroidLibraryPlugin.groovy

```diff
@@ -105,6 +105,7 @@ class AndroidLibraryPlugin implements Plugin<Project> {
                 }
             }
 
+            apply plugin: 'jacoco'
```

Comment:
What about non-android libraries, i.e. plain java. Should this jacoco configuration also be part of `LibraryPlugin`?

Reply:
No, I don't think so. Only the Android plugin uses jacoco so this shouldn't be needed for non-android.